discovery/db_discovery.go

```diff
@@ -41,6 +42,20 @@ func NewDBOrchestratorPoolCache(node *core.LivepeerNode) *DBOrchestratorPoolCach
 	return &DBOrchestratorPoolCache{node: node}
 }
 
+func (dbo *DBOrchestratorPoolCache) GetURLs() []*url.URL {
+	orchs, err := dbo.node.Database.SelectOrchs()
+	if err != nil || len(orchs) <= 0 {
+		return nil
+	}
+
+	var uris []*url.URL
+	for _, orch := range orchs {
+		uri, _ := url.Parse(orch.ServiceURI)
```

Comment:
probably should check for an error and `continue` if there's something wrong; otherwise the url will be empty

Reply:
Fixed